src/plugin.ts

```diff
@@ -72,6 +73,14 @@ export class GltfPlugin extends Evented<GltfPluginEventTable> {
             this.addThreeJsLayer();
             this.initEventHandlers();
         });
+
+        if (pluginOptions?.floorsControl) {
+            const position =
+                typeof pluginOptions.floorsControl === 'boolean'
```

Comment:
Тут ещё можно чуть упростить:

```suggestion
                pluginOptions.floorsControl === true
```

Reply:
а если === 'false' ?

Reply:
а типа выше уже есть проверка что он есть и false быть не может - но блин это так читается не понятно и зависимость от ифа выше... нее